crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/ContributionRulesService.java

```diff
@@ -50,6 +28,9 @@ public CrownCourtOutcome getActiveCCOutcome(List<ApiCrownCourtOutcome> crownCour
     public boolean isContributionRuleApplicable(CaseType caseType,
                                                 MagCourtOutcome magCourtOutcome,
                                                 CrownCourtOutcome crownCourtOutcome) {
-        return getContributionVariation(caseType, magCourtOutcome, crownCourtOutcome).isPresent();
+
+        return (CaseType.EITHER_WAY == caseType) && (crownCourtOutcome == null) &&
```

Comment:
Minor: I think we can drop some of the parentheses here and still achieve the same logic.